app/frontend.py

```
@@ -3,48 +3,139 @@
-            out = resp.json()
-            st.write(f"Fraud Score: {out.get('fraud_score', 'N/A')}")
-            st.write(f"Result: {out.get('result', 'N/A')}")
+with tab4:
+    st.header("Check Combined")
+    combo_email = st.text_area("Email text for combined check:")
+    combo_audio = st.file_uploader("Audio file for combined check:", type=["wav"], key="combo_audio")
+    if st.button("Check Combined"):
+        if combo_email and combo_audio:
+            files = {"file": (combo_audio.name, combo_audio, "audio/wav")}
+            data = {"text": combo_email}
+            resp = requests.post("http://127.0.0.1:8000/check_combined", data=data, files=files)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)